config/eks.js

```diff
@@ -0,0 +1,78 @@
+const { merge } = require('webpack-merge');
```

Comment:
Is there a better name we can come up with other than `eks`? I'm not a huge fan of putting that in our configs.

Reply:
Maybe `dynamic`?